### PR TITLE
Cleanup action

### PR DESCRIPTION
**Pull request by snehasreeramini**
Installs gh and s3cmd.
Cleans the cached objects and s3 cache.

### CODE REVIEW

File: clean_cache/action.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: 'clean_cache'
+description: 'clean the cache'
+inputs:
+  s3cmd:
```
Comment (joernott) [inline clean_cache/action.yml:4]: Where in this action is this parameter used. And what do you want to accomplish with it?

File: clean_cache/action.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  cache_objects:
+    type: string
+    required: true
+    description: 'objects to delete for cleanup'
+
+runs:
+  using: composite
+  steps:
+    - name: Install s3cmd and gh
+      if: ${{ inputs.install_ghcli == 'true' }}
```
Comment (joernott) [inline clean_cache/action.yml:29]: What does this line do? Do we need it?

File: clean_cache/action.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            ID_LIST="$(${GH} actions-cache list --key ${CACHE}|awk '{print $1}'||true)"
+              for ID in ${ID_LIST}; do
+              echo -e "\033[0;35m###  Cleaning cached object '${ID}' from github ###\033[0m"
+                ${GH} actions-cache delete "${ID}" --confirm
+              done
+            done
+          fi
+
+    - name: Clear s3 cache
+      if: ${{ inputs.runs_on != '"ubuntu-latest"' }}
```
Comment (joernott) [inline clean_cache/action.yml:77]: Where is inputs.runs_on defined?

**Review by joernott — CHANGES_REQUESTED** [review]
I've put a few questions here. Let's talk about them tomorrow.

File: clean_cache/action.yml (hunk 1 above)
Comment (snehasreeramini) [reply clean_cache/action.yml]: its wrong

File: clean_cache/action.yml (hunk 2 above)
Comment (snehasreeramini) [reply clean_cache/action.yml]: if condition is not required here as we are not giving install_ghcli as input parameter

File: clean_cache/action.yml (hunk 3 above)
Comment (snehasreeramini) [reply clean_cache/action.yml:72]: it is defined in universal workflow but now used in clear_cache too

**Review by joernott — APPROVED** [review]

**Comment by joernott** [comment]
Is being released together with the merge stuff, closing this PR

**Pull request closed without merging**